docs/source/models/cause.rst

```diff
@@ -275,14 +273,155 @@ Birth Prevalence
+denominator of incidence measures is somewhat more complex and is critical to 
+ensuring an accurate data source to inform cause model transition rates. 
+
+Two incidence measures relevant to cause model transition rate data sources 
+using GBD results are discussed in this section, including measures we refer 
+to as **incidence in the total population** (as estimated by the GBD study) 
+and **incidence in the susceptible** (or *at-risk*) **population.** These 
+measures are defined using the following key concepts:
+
+  **Person-time:** person-time is a measure of the number of individuals 
+  multiplied by the amount of time they individually occupy the population 
+  of interest. Notably, the population of interest varies depending on context 
+  and can be defined by age group, sex, location, time, disease status, etc.
+
+    For example, if one individual is occupies the population of interest for 
+    two years, they contribute two person-years. If another individual is in 
+    our population of interest for 6 months, they contribute 0.5 person-years.
+    Together, these two individuals contribute a total of 2.5 person-years.
+
+  **Susceptible or At-Risk Population:** the susceptible population, also 
+  referred to as the at-risk population, is defined as the population that *
+  does not* have the condition of interest; in other words, the susceptible 
+  population that is at risk of developing the condition. Notably, the number 
+  of individuals in this population will change over time as the following 
+  events occur:
+
+     - Members of the at-risk population develop the condition and are no longer susceptible
+     - Members of the at-risk population die and are no longer susceptible 
+     - Individuals are born or age into the at-risk population and become susceptible
+     - Individuals age out of the at-risk population and are no longer susceptible
+     - Individuals with the condition recover from the condition and re-enter the at-risk population as susceptible (in the case of conditions with remission)
+
+**Total Population Incidence Rate** is estimated by the Global Burden of 
+Disease Study by estimating the number of incident cases that occur in one 
+year and scaling this value per 100,000 individuals of a specified population.
+
+.. math::
+
+  \frac{n_\text{incident cases}}{\text{person-time}_\text{total population}}
+
+Because the denominator of this measure is not specific to a particular cause 
+model state, it is **not** an appropriate data source for cause model 
+transition rates between states. 
+
+.. note:: 
+
+  GBD estimates of total population incidence rate require transformation 
+  prior to use as a cause model transition probability data source (see below 
+  for more detail).
+
+**Susceptible/At-Risk Population Incidence Rate** as discussed here is also 
+referred to as incidence density rate, person-time incidence rate, and in some 
```

Comment:
Both this and the total population incidence rate are technically incidence density rates.  That term just means you have person-time in the denominator.

Reply:
I think *incidence rate* and *incidence density rate* (or *incidence density*) are now essentially treated as synonyms. See e.g.:

http://www.sjsu.edu/faculty/gerstman/hs161/pre-inc-key-points.PDF

https://en.wikipedia.org/wiki/Incidence_(epidemiology)

https://www.investopedia.com/terms/i/incidence-rate.asp

I like the sound of "incidence rate" better than the other two options for general use, but it is good that we note the alternative terminology. In fact, when we say "... and in some cases may simply be referred to as the incidence rate," it may be more accurate to say that *usually*, the term "incidence rate" means the measure that we are calling the "susceptible population incidence rate."

Also note that in the last item of the first link above, they say that at-risk person time can be approximated by *(average population) x (time interval)*. It looks like they mean "average total population," which means that using this measure in the denominator would actually compute what we are calling the "total population incidence rate" that the GBD uses. So perhaps this is a common way to estimate the susceptible incidence rate when total population is the only data available?